data/json/recipes/recipe_others.json

```diff
@@ -3054,18 +3059,34 @@
     "qualities": [ { "id": "DIG", "level": 2 } ],
     "components": [ [ [ "bag_canvas", 1 ] ], [ [ "material_gravel", 2700 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "ACTIVE_EXERCISE",
+    "result": "anvil_crude",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "20 h",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "GRIND", "level": 2 }, { "id": "ANVIL", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ], [ "casting_mold", -1 ] ], [ [ "forge", 200 ], [ "oxy_torch", 200 ] ] ],
```

Comment:
If this is supposed to represent forge welding a bunch of chunks together then the crucible is a distraction. If you're imagining a process where you start with iron chunks and make regular ingots out of them with casting and forge weld them together, then make the input be ingots and we can have a separate recipe for that.
Also, if we're representing forge welding 27 kg of iron together these requirements are not representative of the properties of a typical forge. It's not going to have either the volume or heat capacity for the task you're describing here.
Finally, forge welding requires forging. There's an anvil 1 requirement, but that is intended for a small but irregular forging task. You need something to act as a "large crude anvil".

Reply:
I agree the crucible is a distraction, but every single metalworking recipe wants to use the crucible. It would it easier to get the steel into convenient sizes, but hardly necessary. I'm happy to drop it, just assumed it was policy to keep it for anything involving metal.

As to the forge, keep in mind that you are not heating the entire anvil in the forge. You heat each part and add it, so each chunk of steel is heated then added to the existing part. The 'welding' is hammer welding, where you hammer metal while red-hot and it welds together.

https://web.archive.org/web/20160303175903/http://www.abana.org/downloads/controlled_hand_forging/CHF_10.pdf

The kicker to this method is that you can work with small parts of iron and 'build' the anvil without needing to reheat the entire thing at any point. It's ideal for exactly the situation you describe, one with a limited forge and limited anvil. The heat capacity...I mean the forge can melt steel for casting, if it can do that, it can heat steel red-hot relatively easily.

If you'd like, we can add in a specific requirement for the 'anvil 1' such as a large rock or steel frame? Would that work? I agree you'd want as large a flat surface as possible.

Reply:
I boosted the time even higher, just to represent the work involved.

Sdded the ability to use a chunks or lumps, respectively. Removed crucible since there is not really a need for it as you pointed out. Also increased forge fuel use since it is a lot of steel.